bin/pygrb/pycbc_grb_inj_finder

```diff
@@ -95,56 +93,101 @@ def read_hdf5_triggers(inputfiles, verbose=False):
     outputfile : `str`
         the path of the output HDF5 file to write
     """
-    datasets = {}
+    datasets_n = {}
+    datasets_ifo = {}
 
-    def _scan_dataset(name, obj):
-        if not name.startswith("network") or not isinstance(obj, h5py.Dataset):
+    def _scan_dataset_ifo(name, obj):
+        if name.startswith("network") or not isinstance(obj, h5py.Dataset) or "search" in name:
             return
         if NETWORK_IFO_EVENT_ID_REGEX.match(name):
             return
+        
         shape = obj.shape
         dtype = obj.dtype
         try:
-            shape = numpy.sum(datasets[name][0] + shape, keepdims=True)
+            shape = numpy.sum(datasets_ifo[name][0] + shape, keepdims=True)
         except KeyError:
             pass
         else:
-            assert dtype == datasets[name][1], (
+            assert dtype == datasets_ifo[name][1], (
                 "Cannot merge {0}/{1}, does not match dtype".format(
                     obj.file.filename, name,
                 ))
-        datasets[name] = (shape, dtype)
+        datasets_ifo[name] = (shape, dtype)
+        
+    def _scan_dataset_network(name, obj):
+        if not name.startswith("network") or not isinstance(obj, h5py.Dataset) or "search" in name:
+            return
+        if NETWORK_IFO_EVENT_ID_REGEX.match(name):
+            return
+        shape = obj.shape
+        dtype = obj.dtype
+        try:
+            shape = numpy.sum(datasets_n[name][0] + shape, keepdims=True)
+        except KeyError:
+            pass
+        else:
+            assert dtype == datasets_n[name][1], (
+                "Cannot merge {0}/{1}, does not match dtype".format(
+                    obj.file.filename, name,
+                ))
+        datasets_n[name] = (shape, dtype)
 
     # get list of datasets
-    datasets = {}
+    datasets_n = {}
+    datasets_ifo = {}
     for filename in inputfiles:
         with h5py.File(filename, 'r') as h5f:
-            h5f.visititems(_scan_dataset)
+            h5f.visititems(_scan_dataset_ifo)
+            h5f.visititems(_scan_dataset_network)
 
     position = defaultdict(int)
-
-    out = {}
-
-    # create datasets
-    for dset, (shape, dtype) in datasets.items():
-        # 8: chops off network/ from the dset string
-        out[dset[8:]] = numpy.empty(shape, dtype=dtype)
+    
+    out_dict = {}
+    for x in trig_dict:
+        out_dict[x] = {}
+        
+    placeholder = 0
+
+    # create datasets    
+    for dset, (shape, dtype) in datasets_ifo.items():
+        # 3: chops off H1/ or L1/ or V1/ from the dset string
+        for x in trig_dict:
+            if placeholder == 0 and x!= 'network':
+                placeholder = x
+            if dset.startswith(x):
+                out_dict[x][dset[3:]] = numpy.empty(shape, dtype=dtype)
+            
+    for dset, (shape, dtype) in datasets_n.items():
+        out_dict['network'][dset[8:]] = numpy.empty(shape, dtype=dtype)
```

Comment:
As you did for `H1/`, etc. please add a comment to remind us that the 8 skips `network/`